### DISCUSSION TITLE
Cannot change TRANS properties in 24.3.2

### DISCUSSION THREAD

**Discussion by bnilsson11** [ra3xdh/qucs_s#1029] [Q&A] [answered]
It is not possible to disable initial DC OP in simulation TRANS Properties to see the circuit startup behaviour in qucs-s 21.3.2.
in fact, it is not possible to change anything at all, including "display in schematic".
This works as it should in qucs-s 24.3.1.

Platform:
MacStudio M2
MacOS 15.0.1 (24A348)

**Comment by ra3xdh** [comment, marked as answer]
This issue is already reported and fixed. See #968 and #969. Compile the application from Git or wait for the next release. I am planning to prepare a release package in the next week.
